### PR TITLE
Run test kitchen on Circle-CI (AO-15194)

### PR DESCRIPTION
**Pull request by d-maslyk**
Changes:

- run test kitchen with dokken driver on circle
- Test package installation on Linux
- verify ubuntu 16.04, 18.04, 20.04, centos7, debian8 debian9, debian10
- supress chef infra from printing information about generated templates - to protect secrets

### CODE REVIEW

File: test/integration/default/snap_test.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+describe file('/opt/SolarWinds/Snap/etc/plugins.d/publisher-processes.yaml') do
+  it { should exist }
+  it { should be_owned_by 'solarwinds' }
+  its('group') { should eq 'solarwinds' }
+  its('mode') { should cmp '0640' }
+  its('content') { should match(%r{hostname_alias: test-kitchen}) }
+end
+
+describe file('/opt/SolarWinds/Snap/etc/plugins.d/publisher-appoptics.yaml') do
```
Comment (lootek) [inline test/integration/default/snap_test.rb:26]: please add checks for default tasks (tasks-autoload.d/)

**Review by lootek — APPROVED** [review]
LGTM

**Comment by lootek** [comment]
Also, please btw update codeowners file to be the same as https://github.com/librato/snap/blob/master/.github/CODEOWNERS

File: test/integration/default/snap_test.rb (hunk 1 above)
Comment (d-maslyk) [reply test/integration/default/snap_test.rb:27]: Added

**Comment by d-maslyk** [comment]
> Also, please btw update codeowners file to be the same as https://github.com/librato/snap/blob/master/.github/CODEOWNERS

Done.

**Review by lootek — APPROVED** [review]

**Review by tomekkolo — APPROVED** [review]

**Pull request merged**